scripts/release/packager/project.yml

```diff
@@ -18,7 +18,7 @@ targets:
         DEFINES_MODULE: "YES"
         PRODUCT_NAME: $(TARGET_NAME:c99extidentifier)
         CODE_SIGN_STYLE: Automatic
-        SWIFT_TREAT_WARNINGS_AS_ERRORS: "YES"
+        SWIFT_TREAT_WARNINGS_AS_ERRORS: "NO"
```

Comment:
I disabled this as it was causing the `build-sdk-for-api-compatibility-check` step to fail  https://app.circleci.com/pipelines/github/mapbox/mapbox-maps-ios/6562/workflows/5e6fe8c2-94ba-4bc2-831c-4529b57e322f/jobs/61993

Reply:
I don't see any other way around this